src/tito/tagger/main.py

```diff
@@ -405,7 +405,7 @@ def _bump_version(self, release=False, zstream=False, force=False):
                                         increase_zstream(match.group(2)),
                                         "\n"
                         ))
-                elif force:
+                elif self._use_version is not None:
```

Comment:
If people were able to over-ride the `ForceVersionTagger` in downstream implementations of custom taggers, however, we need to keep that class around (with no methods?) so we don't break them.